### DISCUSSION TITLE
fix-admin-consent-sp-token-retrieval-flows-spfx/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#623] [Comments]
# fix-admin-consent-sp-token-retrieval-flows-spfx/

Learn how to fix admin consent issues with the  00000003-0000-0ff1-ce00-000000000000 SharePoint Entra App.

https://www.eliostruyf.com/fix-admin-consent-sp-token-retrieval-flows-spfx/

**Comment by jcoolsen** [comment]
Thank you for this. It matched an issue with a client (they only had the 00000003-0000-0ff1-ce00-000000000000 client listed) and the solution worked perfectly. It boggles my mind how this data can get messed up.

**Reply by parmira5** [reply]
Can we call the “Microsoft.SharePoint.Internal.ClientSideComponent.Token.AcquireOBOToken” endpoint directly to fetch a token in our custom solution? Having an issue with Safari where ITP is enabled, popups are blocked, and the customer hates the redirect.
